src/main/java/mate/academy/bookshop/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.bookshop.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import mate.academy.bookshop.validation.PasswordMatch;
+
+@Data
+@PasswordMatch

```

Comment:
let's not make it just for passwords, but universal, so that we could use it with different classes and fields, using reflection, passing field names as arguments to that annotation, and renameing it to some `@FieldsMatch`